charts/posthog/templates/_helpers.tpl

```diff
@@ -132,6 +132,20 @@ Set postgres URL
     postgres://{{- .Values.postgresql.postgresqlUsername -}}:{{- template "posthog.postgresql.password" . -}}@{{- template "posthog.postgresql.host" .  -}}:{{- template "posthog.postgresql.port" . -}}/{{- .Values.postgresql.postgresqlDatabase }}
 {{- end -}}
 
+{{/*
+Set zookeeper host
+*/}}
+{{- define "posthog.zookeeper.host" -}}
+    {{- template "posthog.fullname" . }}-posthog-zookeeper
```

Comment:
doesn't `posthog.fullname` already contain `posthog`?

Reply:
It does, but if you look at how we name all of the dependent services they end up `posthog-posthog-<service>` which is the `<deployment>-posthog-<service>` which works out, though a bit repetitive usually.